patroni/postgresql/config.py

```diff
@@ -480,13 +480,19 @@ def primary_conninfo_params(self, member):
             ret['krbsrvname'] = self._krbsrvname
         if 'dbname' in ret:
             del ret['dbname']
+        # Add target_session_attrs in case more than one hostname is specified
+        # (libpq client-side failover) making sure we hit the primary
+        if not 'target_session_attrs' in ret and self._postgresql.major_version >= 100000:
```

Comment:
```
E713 test for membership should be 'not in'
```

and the same for https://github.com/zalando/patroni/pull/2193/files#diff-4ff0e58c1acc3485e8ceb3239cb81a7ad629c6a0c2c8d43f291509658372e645R159

Reply:
Fixed in 08f8d8d